src/Http/PendingRequest.php

```diff
@@ -193,6 +194,9 @@ protected function mergeRequestProperties(): static
             ->merge($connector->middleware())
             ->merge($request->middleware());
 
+        $delay = $request->delay()->all();
+        $this->delay()->set($delay);
```

Comment:
@anthonyvancauwenberghe Do you need these two lines:

```php
$delay = $request->delay()->all();
$this->delay()->set($delay);
```
Because you have `mergeDelay` above?